src/main/resources/db/migration/postgres/V2.4.2.5__file_value_encode_sql_function.sql

```diff
@@ -0,0 +1,32 @@
+CREATE OR REPLACE FUNCTION encode_file_value(text)
+    RETURNS text
+    AS $$
+    -- Function to encode a URI component
+    SELECT
+        string_agg(
+             -- Bytes <= 1 are safe because they are part of the standard ASCII chars which don't need to be checked
+            CASE WHEN bytes > 1
+                -- Skip encoding of these characters
+                -- - Alphanumeric characters: `0-9`, `a-z`, `A-Z`
+                -- - Unreserved characters: `-`, `_`, [`.`](command:_github.copilot.openRelativePath?%5B%7B%22scheme%22%3A%22file%22%2C%22authority%22%3A%22%22%2C%22path%22%3A%22%2FUsers%2Fbbolt%2Fschrodinger%2FACAS-753-url-enncode-file-values%22%2C%22query%22%3A%22%22%2C%22fragment%22%3A%22%22%7D%5D "/Users/bbolt/schrodinger/ACAS-753-url-enncode-file-values"), `~`
```

Comment:
@brianbolt there's a weird link to a local filepath on this line that looks like an accidental inclusion

Reply:
Thanks. Fixed.